pkg/events/derive/hooked_seq_ops.go

```diff
@@ -19,18 +37,18 @@ func deriveHookedSeqOpsArgs(kernelSymbols *helpers.KernelSymbolTable) deriveArgs
 		if err != nil || len(seqOpsArr) < 1 {
 			return nil, err
 		}
-		seqOpsName := utils.ParseSymbol(seqOpsArr[0], kernelSymbols).Name
-		hookedSeqOps := make([]trace.HookedSymbolData, 0)
-		for _, addr := range seqOpsArr[1:] {
+		hookedSeqOps := make(map[string]trace.HookedSymbolData, 0)
+		for i, addr := range seqOpsArr {
 			inTextSegment, err := kernelSymbols.TextSegmentContains(addr)
 			if err != nil {
 				continue
 			}
 			if !inTextSegment {
 				hookingFunction := utils.ParseSymbol(addr, kernelSymbols)
-				hookedSeqOps = append(hookedSeqOps, trace.HookedSymbolData{SymbolName: hookingFunction.Name, ModuleOwner: hookingFunction.Owner})
+				hookedSeqOps[NetSeqOps[i/4]+"_"+NetSeqOpsFuncs[i%4]] =
```

Comment:
Could you make this indexing more concise/human-readable (maybe a "translation" function)